src/aff_expr.jl

```diff
@@ -150,9 +150,11 @@ end
 Base.zero(::Type{GenericAffExpr{C,V}}) where {C,V} = GenericAffExpr{C,V}(zero(C), OrderedDict{V,C}())
 Base.one(::Type{GenericAffExpr{C,V}}) where {C,V}  = GenericAffExpr{C,V}(one(C), OrderedDict{V,C}())
 Base.zero(a::GenericAffExpr) = zero(typeof(a))
-Base.one( a::GenericAffExpr) =  one(typeof(a))
+Base.one(a::GenericAffExpr) = one(typeof(a))
 Base.copy(a::GenericAffExpr) = GenericAffExpr(copy(a.constant), copy(a.terms))
 Base.broadcastable(a::GenericAffExpr) = Ref(a)
+Base.getindex(a::GenericAffExpr{C,V}, v::V) where {C,V} = get(a.terms, v, zero(V))
```

Comment:
`V` is the variable type and `C` is the coefficient type, so if our `get` fails, the default should be `zero(C)`, not `zero(V)`: `get(a.terms, v, zero(C))`.